defaults/capabilities.go

```diff
@@ -0,0 +1,160 @@
+	// 	 outside the range 0 to 6), SO_RCVBUFFORCE, and SO_SNDBUFFORCE.
+	CapNetAdmin = "CAP_NET_ADMIN"
+
+	// CapNetBindService - Bind a socket to Internet domain privileged ports (port numbers less than 1024).
+	CapNetBindService = "CAP_NET_BIND_SERVICE"
+
+	// CapNetBroadcast - Make socket broadcasts, and listen to multicasts.
+	CapNetBroadcast = "CAP_NET_BROADCAST"
+
+	// CapNetRaw -
+	// * Use RAW and PACKET sockets
+	// * bind to any address for transparent proxying.
+	CapNetRaw = "CAP_NET_RAW"
+
+	// CapSetgid -
+	// * Make arbitrary manipulations of process GIDs and supplementary GID list
+	// * forge GID when passing socket credentials via UNIX domain sockets
+	// * write a group ID mapping in a user namespace
+	CapSetgid = "CAP_SETGID"
+
+	// CapSetfcap - Set file capabilities.
+	CapSetfcap = "CAP_SETFCAP"
+
+	// CapSetpcap -
+	// * If file capabilities are not supported: grant or remove any  capability in the caller's permitted
+	// 	   capability set to or from any other process.
+	// *  If file capabilities are supported: add any capability from  the calling thread's bounding set to its
+	//     inheritable set; drop capabilities from the bounding set; make changes to the securebits flags.
+	CapSetpcap = "CAP_SETPCAP"
+
+	// CapSetuid -
+	// * Make arbitrary manipulations of process UIDs
+	// * forge UID when passing socket credentials via UNIX domain sockets
+	// * write a user ID mapping in a user namespace
+	CapSetuid = "CAP_SETUID"
+
+	// CapSysAdmin - Perform administrative operations on the system (see man capabilities(7))
+	CapSysAdmin = "CAP_SYS_ADMIN"
+
+	// CapSysBoot - Use reboot and kexec_load.
+	CapSysBoot = "CAP_SYS_BOOT"
+
+	// CapSysChroot - Use chroot.
+	CapSysChroot = "CAP_SYS_CHROOT"
+
+	// CapSysModule - Load and unload kernel modules.
+	CapSysModule = "CAP_SYS_MODULE"
+
+	// CapSysNice -
+	// * Raise processes nice value
+	// * set real-time scheduling policies for processes
+	// * set CPU affinity for arbitrary processes
+	// * set I/O scheduling class and priority for arbitrary processes
+	CapSysNice = "CAP_SYS_NICE"
+
+	// CapSysPacct - Use acct.
+	CapSysPacct = "CAP_SYS_PACCT"
+
+	// CapSysPtrace - Trace, inspect and modify the state of arbitrary processes.
+	CapSysPtrace = "CAP_SYS_PTRACE"
+
+	// CapSysRawio - Perform various privileged IO operations (see man capabilities(7))
+	CapSysRawio = "CAP_SYS_RAWIO"
+
+	// CapSysResource - Perform various privileged resource configuration operations (see man capabilities (7))
+	CapSysResource = "CAP_SYS_RESOURCE"
+
+	// CapSysTime - Set system and hardware clocks.
+	CapSysTime = "CAP_SYS_TIME"
+
+	// CapSysTtyConfig -
+	// * Use vhangupl
+	// * Perform various privileged ioctl operations on TTYs
+	CapSysTtyConfig = "CAP_SYS_TTY_CONFIG"
+
+	// CapSyslog -
+	// * Perform privileged syslog operations
+	// * View kernel addresses exposed via /proc under certain conditions
+	CapSyslog = "CAP_SYSLOG"
+
+	// CapWakeAlarm - trigger something that will wake up the system
```

Comment:
these descriptions are awesome.  Did you write them yourself or get them elsewhere?